src/Char.elm

```diff
@@ -16,25 +16,31 @@ module Char
 
 import Native.Char
 
+isBetween : Char -> Char -> Char -> Bool
+isBetween low high char = 
+    let code = toCode char
+    in  (code >= toCode low) && (code <= toCode high)
```

Comment:
The Basics module isn't auto-imported here, so (&&), (>=), and (<=) are undefined.

Is it safe to `import Basics (..)`?  I assume the reason to not do so is to avoid circular dependencies...?  If that's the case, is it uncouth to use `Native.Basics.and`, etc?